app/ts/components/App.tsx

```diff
@@ -1,10 +1,47 @@
-import { Route, Router } from './Router.js'
-import { SendEthPage } from './SendEthPage.js'
+import { Route, Router, useParams } from './HashRouter.js'
 
 export function App() {
 	return (
 		<Router>
-			<Route path='/' component={SendEthPage} />
+			<Route path='#send' component={Send} />
```

Comment:
```suggestion
			<Route path = '#send' component = { Send } />
```

did we disagree on styling here? 🤔

Reply:
For TSX I prefer less spaces than in TS.

Reply:
😬